plugins/module_utils/network/sonic/config/logging/logging.py

```diff
@@ -393,6 +418,10 @@ def get_delete_requests(self, configs, delete_all):
             if servers_request:
                 requests.extend(servers_request)
 
+        if (configs.get('security_profile') or delete_all) and (configs.get('security_profile') is not None):
```

Comment:
The condition could be simplied as below, since deletion is performed only when 'security_profile' is not 'None' in configs.
```suggestion
        if configs.get('security_profile'):
```

Reply:
This would lead to failure of delete all n and overridden cases hence handling it this way. The request flows for normal delete , Delete all ( no input config being present), and over ridden cases ( where input has remote servers but not security_profile) and also to handle idempotency failures

Reply:
Addressed